client/my-sites/sidebar-unified/style.scss

```diff
@@ -196,6 +196,46 @@ $font-size: rem( 14px );
 				}
 			}
 		}
+
+		.notice {
+			background-color: #3c434a;
```

Comment:
This color is used twice, it should probably be a variable.

Reply:
Done!